assets/templates/partials/sort.tmpl

```diff
@@ -6,10 +6,21 @@
 {{ range $theFilter := .Data.Filter }}
     <input type="hidden" name="filter" value="{{ $theFilter }}">
 {{end}}
-<!-- rss feed here -->
-<div class="ons-field--search-sort search__sort__select">
-    <label for="sort" class="ons-label sort__label font-size--16">Sort by</label>
-    <select class="ons-input ons-input--select ons-input--sort-select input select font-size--16" id="sort" name="sort">
+
+<div class="ons-grid ons-grid--flex@m ons-u-pb-s">
+  <div class="ons-grid__col ons-u-wa--">
+    <div class="ons-field--inline">
+      <label
+        class="ons-label"
+        for="select-calendar-item-order"
```

Comment:
The `for` should be the `id` of the select i.e. `sort` or update the sort id to match this